doc/v2/documentation_data_format.rst

```diff
@@ -620,7 +642,69 @@ Detailed field description
 
   Prior parameters used for the objective function during estimation.
   For more detailed documentation, see ``initializationPriorParameters``.
+  As for initialization priors, truncated priors are allowed.
+
+Prior distributions
+~~~~~~~~~~~~~~~~~~~
 
+For ``initializationPriorParameters`` and ``objectivePriorType`` several
```

Comment:
```suggestion
For ``initializationPriorType`` and ``objectivePriorType`` several
```

or did you mean `*PriorParameters` for both? In any case, I guess one needs to be changed.